### ISSUE TITLE
Google calendar will not ask to generate token

### ISSUE THREAD

**Issue by IronWolfMC** [pimutils/vdirsyncer#414]
vdirsyncer version: 0.9.4.dev28+ngf8a3961 (Apr 7)
Python: 2.7.6
OS: Ubuntu 14.04 Server (Docker)

When attempting to use google_calendar, vdirsyncer merely errors instead of requesting to generate a google token as the documentation states.

Configuration:
```
[general]
status_path = ~/.config/vdirsyncer/status/

[pair my_calendars]
a = my_cal
b = my_gmail
collections = null

[storage my_gmail]
type = google_calendar
token_file = ~/.google-token

[storage my_cal]
type = caldav
url = http://my.owncloud.server/cloud/remote.php/caldav/calendars/USERNAME/personal
username = USERNAME
password = ****************

```

Error: (Gist: https://gist.github.com/IronWolfMC/46b7a6a80a5fd6fbacd8aff6924ed6bd)
```
root@78fc4b3c9c65:~/.config/vdirsyncer# vdirsyncer sync
warning: Python 2 support will be dropped. Please switch to at least Python 3.3 as soon as possible. See https://github.com/pimutils/vdirsyncer/issues/219 for more information.
Syncing my_calendars
Unhandled exception occured for my_calendars.
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/vdirsyncer-0.9.4.dev28+ngf8a3961-py2.7.egg/vdirsyncer/cli/tasks.py", line 61, in sync_collection
    a = storage_instance_from_config(collection.config_a)
  File "/usr/local/lib/python2.7/dist-packages/vdirsyncer-0.9.4.dev28+ngf8a3961-py2.7.egg/vdirsyncer/cli/utils.py", line 385, in storage_instance_from_config
    return handle_storage_init_error(cls, new_config)
  File "/usr/local/lib/python2.7/dist-packages/vdirsyncer-0.9.4.dev28+ngf8a3961-py2.7.egg/vdirsyncer/cli/utils.py", line 376, in storage_instance_from_config
    return cls(**new_config)
  File "/usr/local/lib/python2.7/dist-packages/vdirsyncer-0.9.4.dev28+ngf8a3961-py2.7.egg/vdirsyncer/storage/google.py", line 133, in __init__
    **kwargs
  File "/usr/local/lib/python2.7/dist-packages/vdirsyncer-0.9.4.dev28+ngf8a3961-py2.7.egg/vdirsyncer/storage/dav.py", line 701, in __init__
    super(CaldavStorage, self).__init__(**kwargs)
  File "/usr/local/lib/python2.7/dist-packages/vdirsyncer-0.9.4.dev28+ngf8a3961-py2.7.egg/vdirsyncer/storage/dav.py", line 374, in __init__
    self.session_class.init_and_remaining_args(**kwargs)
  File "/usr/local/lib/python2.7/dist-packages/vdirsyncer-0.9.4.dev28+ngf8a3961-py2.7.egg/vdirsyncer/storage/dav.py", line 301, in init_and_remaining_args
    return cls(**self_args), remainder
  File "/usr/local/lib/python2.7/dist-packages/vdirsyncer-0.9.4.dev28+ngf8a3961-py2.7.egg/vdirsyncer/storage/google.py", line 50, in __init__
    with open(token_file) as f:
IOError: [Errno 2] No such file or directory: '/root/.google-token'
error: 1 out of 2 tasks failed.
```

**Comment by untitaker** [comment]
Thanks, this is actually a bug in Python 2 only, and will get fixed in the 0.10 release with #415. This bug will close automatically once the tests pass and the fix is merged into master.

I strongly recommend using Python 3.3+, as I plan on dropping Python 2 support. If you want a technical overview/rationale, see #219

**Comment by untitaker** [comment]
My builds are too slow! I merged this manually.

Also, if you installed this via pip, there's usually a `pip2` and a `pip3` in your PATH. Also `virtualenv2` vs `virtualenv3` etc...
